Packs/BitwardenPasswordManager/Integrations/BitwardenPasswordManager/BitwardenPasswordManager.py

```diff
@@ -85,7 +95,7 @@ def test_module(client: Client) -> str:
 
 
 def get_events_command(client: Client, args: Dict[str, Any]) -> tuple:
-    limit = args.get("limit", DEFAULT_MAX_FETCH)
+    limit = int(args.get("limit", DEFAULT_MAX_FETCH))
```

Comment:
```suggestion
    limit = arg_to_number(args.get("limit", DEFAULT_MAX_FETCH)) or DEFAULT_MAX_FETCH 
```

better practice to when set with 0/ empty value